docs/source/_inc/_hardware-requirements.md

```diff
@@ -0,0 +1,19 @@
+The hardware requirements below give a rough estimate of the minimum hardware setup needed for a Plone server.
+
+A single Plone installation is able to run many Plone sites.
+
+-   Installation of the Plone backend and Classic UI frontend requires a minimum of 256 MB of RAM and 2GB of disk swap space.
+-   Installation of the Seven frontend requires a minimum of 2GB of RAM.
```

Comment:
in branch main it's still 'Volto', not 'Seven', isn't it?

Reply:
Done in https://github.com/plone/volto/pull/6925